cogs/verification.py

```diff
@@ -0,0 +1,25 @@
+import discord
+from discord.ext import commands
+
+class Verification(commands.Cog):
+    """ The TPK Verification System | Developed by DuckMasterAl """
+    def __init__(self, bot):
+        self.bot = bot
+        self.channel = 123# Verification Channel ID
+        self.message = 123# Verification Message ID
+
+    @commands.Cog.listener('on_raw_reaction_add')
+    async def reaction(self, payload):
+        """ Checks for a Reaction on the Verification Message """
+        if payload.message_id == self.message:
+            try:
+                await payload.member.send('Thanks for wanting to verify in TPK [...]')# wait_for
```

Comment:
@BluewyDev what should the bot send to the user after they react?
Should it ask a question, or should it just say "Your verification request will be looked at shortly by a staff member"?

Also, are we pinging the staff team (or a randomly selected online staff member) for every new verification request?

Reply:
The bot should ask questions first.

Reply:
But what should the questions be?
> Also, are we pinging the staff team (or a randomly selected online staff member) for every new verification request?